.github/workflows/build-on-windows.yml

```diff
@@ -0,0 +1,286 @@
+#
+# .github/workflows/build-on-windows.yml
+#
+# Copyright 2021 Jens A. Koch.
+# SPDX-License-Identifier: BSL-1.0
+# This file is part of ttauri-project.
+#
+
+name: "Build on Windows"
+
+on: [push, pull_request]
+
+jobs:
+
+# ---------------------------------------------------------------------------------------
+
+  build-and-test:
+
+# ---------------------------------------------------------------------------------------
+
+    name: ${{ matrix.config.VCPKG_TARGET_TRIPLET }} PCH:${{ matrix.config.TT_BUILD_PCH }}
+    needs: install-vulkan-sdk-and-runtime
+    # https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md
+    runs-on: windows-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - { VCPKG_TARGET_TRIPLET: x64-windows-static, BUILD_SHARED_LIBS: OFF, TT_BUILD_PCH: ON}
```

Comment:
I changed ttauri so that BUILD_SHARED_LIBS is now automatically gets set by the VCPKG_TARGET_TRIPLET.